parser.y

```diff
@@ -3543,10 +3543,12 @@ SimpleExpr:
 |	"CONVERT" '(' Expression "USING" StringName ')'
 	{
 		// See https://dev.mysql.com/doc/refman/5.7/en/cast-functions.html#function_convert
-		charset1 := ast.NewValueExpr($5)
-		$$ = &ast.FuncCallExpr{
-			FnName: model.NewCIStr($1),
-			Args: []ast.ExprNode{$3, charset1},
+		x := types.NewFieldType(mysql.TypeString)
```

Comment:
PTAL @tiancaiamao

Reply:
Ident.

Reply:
It seems a bug. Would you please file a PR to the TiDB repo and cover this bug fix? @buzzers
and PTAL @XuHuaiyu

Reply:
Fix this one `SELECT CONVERT(_latin1'Müller' USING utf8)` in TiDB if you'd like to.
https://github.com/pingcap/tidb/blob/master/expression/builtin_cast.go
A dummy implementation is acceptable as long as the new introduced `CastConvertUsingFunction` doesn't make tidb panic

Reply:
Take a look here https://github.com/pingcap/parser#how-to-update-parser-for-tidb

Reply:
OK, and other problem, `AVG(test_score)` and ``` `AVG`(`test_score`) ```, is this really a bug?

Reply:
`CastConvertUsingFunction` what is it used for, and why do you have this function?

Reply:
@tiancaiamao This modification seems to have led to Tidb's TestInferType test case error, but I do not know how to fix it, can you give me an opinion?
Error out now typeinfer_test.go:459 `{ "convert (c_double_d using c_text_d) ", MySQL. Typelongblob, CharSet. CharsetUTF8MB4, 0, MySQL. Maxblobwidth, types. Unspecifiedlength},`.
Expected RetType not equal.